### ISSUE TITLE
Findable Help SC3.2.6 - available or provided?

### ISSUE THREAD

**Issue by guyhickling** [w3c/wcag#1232]
> "...if one of the following is available..."

In my opinion that "available" in this SC would be better if changed to "provided in the web pages" or "displayed in the web pages" or some similar phrase.

"is available" means something is not necessarily present but it can be obtained, it is available but you might have to do something to get it. In terms of companies and their websites, a company might have a phone number _available_, but if it has not been _provided_ on the website then it does not fall in scope of this SC. Many companies don't show their phone number on their website.

"is provided" means it is already present (i.e. somewhere on the website, in this scenario). 

This is just a small issue, but we shouldn't make readers of anything work harder than they have to.

**Comment by scha14** [comment]
Hi @guyhickling is it okay to close this issue and continue the discussion on https://github.com/w3c/wcag/issues/1242 where all comments are together? Thank you for your patience.

**Comment by scha14** [comment]
Hi @guyhickling is it okay to close this issue and continue the discussion on #1242 where all comments are together? Thank you for your patience.

**Comment by scha14** [comment]
Draft response: 

> The interpretation of available and provided seems consistent with the intent of the SC. _Provided_ would limit the help mechanisms to the ones already present on a site. _Available_ , on the other hand would mean a site provides a help mechanism that can be obtained but is not necessarily present. I see available as a superset of provided, which would allow for more help options on more pages/sites.

**Comment by guyhickling** [comment]
I see where you are coming from in that difference between available and provided. My concern is that this SC may require them (or look as though it requires, which amounts to the same thing) to show a method of contact on the website that they don't want to make available for the public to contact them on. I don't think it should ask websites to show help that isn't  already on the site somewhere. That's a tricky path to go down.

Take a small company providing an online information service that just provides some free information and nothing else. For instance a free weather forecast. Assume they don't have a chat mechanism or any Contact form because they don't feel it is needed, and don't want to be contacted by enquiries because they are providing free info only, on a take-it-or-leave-it basis. Assume also they don't have any FAQs pages or anything like that because everything is sufficiently explained in the website content. 

Out of the 4 help mechanisms that only leaves providing a phone number or email address as the possible source of help. They have a phone number of course, but as stated above they don't necessarily want users asking questions on it. 

But this SC effectively seems to be saying "You have a phone number don't you? So you must provide it on the website whether you like it or not if you want to have WCAG compliance". Obviously that isn't the intention of this SC, but it could be taken that way!

I'm working on the assumption that any company that has a method(s) of contact available that they want people to use, they would _already_ mention it on their website - why wouldn't they? So we don't want this SC to look as though we are telling them to show something they don't want to show! I would rather miss a few edge scenarios through saying "already provided on the site" rather than look like it is demanding something we aren't actually asking for.

That is why I suggested the amendment in this issue. It is covered in the Understanding (where it says "A Web site is not supported"), but I think using "available" in the SC gives the wrong understanding up front. (BTW, I suggest keeping this particular discussion here for the moment, as 1242 is just there as a summary of all the suggested amends.)

**Comment by scha14** [comment]
Draft response 

> I am having a hard time seeing companies interpreting _available_ that way, but perhaps _supported_ would make it clearer. That change is in the PR that changes one > one or more. https://github.com/w3c/wcag/pull/1357 and https://github.com/w3c/wcag/pull/1356

**Comment by alastc** [comment]
> "is available" means something is not necessarily present but it can be obtained, it is available but you might have to do something to get it.

This is true, we need to allow for people to link through to contacts, not just make them present on every page. E.g. if you have a large site with many contact mechansims (numbers, emails, forms etc) then having a link on every page makes sense. We don't want to force all contact mechanism onto every page.


> My concern is that this SC may require them (or look as though it requires, which amounts to the same thing) to show a method of contact on the website that they don't want to make available for the public to contact them on. 

I really struggle to see how people would interpret "if one of the following is available" in that way. If it isn't available (on the website), there is no requirement to add it.

> any company that has a method(s) of contact available that they want people to use, they would already mention it on their website - why wouldn't they? 

The problem this is trying to solve is websites that tuck it away, and don't link to it consistently. That was the core requirment coming from the COGA task force. (Obviously they would rather orgs had to provide help, but this was the feasible version.)

**Comment by guyhickling** [comment]
@scha14  Yes, thank you, "supported" covers it. It conveys the idea that it is a help mechanism supported on the website, which is what I was after. 

I have just looked at yesterday's (3 Sept) new editor's draft and those changes aren't there yet, but I assume they are on the way and will show up in due course.

**Comment by alastc** [comment]
Hi everyone, this was discussed today and the general consensus was that "provided" would be the best term. "Supported" would work well if it was a 'mechanism', but that doesn't work for all the options.

The current proposal is:
> For single page Web applications or any set of Web pages, if one or more of the following ways of finding help is provided, then access to at least one way of finding help is included in the same relative order on each page:

**Comment by scha14** [comment]
Hi @guyhickling , closing this following your comment in https://github.com/w3c/wcag/issues/1231 that changes available to provided.
